### ISSUE TITLE
License file in MANIFEST.in

### ISSUE THREAD

**Issue by gwaybio** [0x00B1/persistence#112]
See https://github.com/conda-forge/staged-recipes/pull/8829#issuecomment-511869498

Good to include the LICENSE.md file in the MANIFEST.in file (see [example](https://github.com/pypa/sampleproject/blob/master/MANIFEST.in))

**Comment by diskontinuum** [comment]
What does "include" mean in the example?  Copy-paste the actual code , i.e. all scripts in the package?!

**Comment by gwaybio** [comment]
Nope, that would be a bad idea! Include meaning this https://github.com/pypa/sampleproject/blob/master/MANIFEST.in#L6-L7
